### PR TITLE
Sync

### PR DESCRIPTION
**Pull request by ghost1face**
These changes should keep this project in sync with the original javascript library.  There were a few cases that were added to the js lib.

Also added the latest commit this lib is synced with, that way it's easier to set expectations and maintain.

### CODE REVIEW

File: Pluralize.NET.Tests/TestData/InputData.csv [hunk 1]
```diff
@@ -592,6 +591,52 @@ accomplice,accomplices
...
+baby,babies
+cookie,cookies
+budgie,budgies
+calorie,calories
+brownie,brownies
+lolly,lollies
+hippie,hippies
+smoothie,smoothies
+techie,techies
+specie,species
```
Comment (sarathkcm) [inline Pluralize.NET.Tests/TestData/InputData.csv]: Is this correct? Shouldn't it be an uncountable?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 
 # What is it?
-This is a C# port of Blake Embrey's [pluralize](https://github.com/blakeembrey/pluralize) library which helps in pluralizing or singularizing any English word.
+This is a C# port of Blake Embrey's [pluralize](https://github.com/blakeembrey/pluralize) library which helps in pluralizing or singularizing any English word. Changes reflect commits up to https://github.com/blakeembrey/pluralize/commit/abb399111aedd1d62dd418d7e0217d85f5bf22c9 from August 27, 2019.
```
Comment (sarathkcm) [inline README.md]: It would be nice to put this as a blockquote. What do you think?

File: Pluralize.NET.Tests/TestData/InputData.csv (hunk 1 above)
Comment (ghost1face) [reply Pluralize.NET.Tests/TestData/InputData.csv]: Let me double check here to make sure.

File: README.md (hunk 2 above)
Comment (ghost1face) [reply README.md]: I agree, changed to use a blockquote here

**Comment by ghost1face** [comment]
I've rolled back the changes here to be in sync up to [May 25, 2019](https://github.com/blakeembrey/pluralize/commit/0265e4d131ecad8e11c420fa4be98b75dc92c33d) commit.  It looks like species was broken in the latest commit, I think we should strive to keep this library updated, however do so for stable releases/tags.

File: Pluralize.NET.Tests/TestData/InputData.csv (hunk 1 above)
Comment (sarathkcm) [reply Pluralize.NET.Tests/TestData/InputData.csv]: I found this discussion in Wiktionary https://en.wiktionary.org/wiki/Talk:specie
Also this
https://brians.wsu.edu/2016/05/31/specie/

**Comment by sarathkcm** [comment]
>I think we should strive to keep this library updated, however do so for stable releases/tags.

Absolutely. 

Please go ahead and merge the pull.

**Comment by sarathkcm** [comment]
> It looks like species was broken in the latest commit

I will raise it in the original library. 

Even with the version that is published in npm, it seems to pluralize the word 'specie' into 'species', which is wrong. Both are separate words, and their plural/singular forms seem to be themselves.

**Pull request merged**